demo_modules/gears/gears.js

```diff
@@ -286,15 +288,35 @@ class GearsClient extends ModuleInterface.Client {
       const baseRadius = baseDiameter / 2;
       const outsideRadius = pitchRadius + addendum;
       const rootRadius = outsideRadius - wholeDepth;
-      
+
+      this.gearDetails_[key] = {
+        pitchDiameter: pitchDiameter,
```

Comment:
You can simply write pitchDiameter here like so:

... = {
  pitchDiameter,
  diametralPitch,
  ...
}